### DISCUSSION TITLE
lets talk fans

### DISCUSSION THREAD

**Discussion by Sonicboom247** [Jyers/Marlin#1844] [Ideas]
ok, i know the whole loud fan, controllable fan subject has been beaten to a pulp but technology changes and options that might not have been available at one time might be now.

So as we know the Ender 3 V2 has FOUR fans, Part Cooling / Hotend / PSU / Motherboard

The part cooling is controllable via gcode so that one is good to go.

The Hotend is hard wired to 24v to always be on, would be nice to have it controllable when extruder is heated? is this even an option on this board?

PSU fan switches on and off once the power supply cover is removed and replaced with one with slots from thingiverse. so that one is good to go.

and lastly the darn motherboard or controller fan...for me (not sure about others) it seems to be wired to my part cooling fan so if i want to print PETG and i want to run my fan barely if at all, then the drivers are cooking.

the only options seem to be 
1. a buck converter straight to 24v and then maybe upgrade the fan to a 50 or 80mm 12v so it runs a bit quieter but constant.
2. simply wire the fan straight to 24v and deal with the noise.
3.  remove the fan altogether and add one massive M2 heatsink to cover all the drivers.

**Comment by Cal-LF** [comment]
There are many fan replacement options for the mainboard. My setup uses 80mm computer with a buck converter. Benefit is adjustable speed/noise. Same buck converter runs a Y fan placed behind and below the stepper. Buck converter is connected to the 24vdc input terminals on the mainboard or can be connected to the PSU 24vdc terminals. Risers are needed for clearance.
The Y fan is from the 60mm PSU and is quite noisy but free.

![Image011](https://user-images.githubusercontent.com/76499230/153714041-04bdf046-6238-44a8-9332-fcb90794497b.jpg)
.

**Reply by Sonicboom247** [reply]
I'm really digging that setup, I added a heatsink to my Y stepper and I need to find my laser to get a temp reading but i can feel the heatsink pretty hot and stepper not so much anymore, i still need to dial in the Vref for it but might leave it as is for a dual Z upgrade, but that's for another discussion. maybe a nice 120mm blower setup with a Y..
I have a ton of 12v PC fans ranging from  40mm plain black up to 120mm Noctua with a few RGB ones in the mix.  I might have to play with this idea and strip my XYZ Davinci Jr of its Noctua upgrades..

**Reply by Sonicboom247** [reply]
Liked it so much i found my other buck converter and fan from one of my OLD pc's, did a bit of research and fan should be ok anywhere under 14.5v, good thing is i can still adjust it from underneath.
Now to figure out how to wire it so it's not being controlled by the parts fan...
![20220212_200749](https://user-images.githubusercontent.com/84334417/153765311-f86195e9-c6c3-456e-9d48-12db4f34a2b6.jpg)
![20220213_114132](https://user-images.githubusercontent.com/84334417/153765314-3383ef30-bc43-4036-a4e8-a56acc0511a9.jpg)

**Reply by BlueWings172** [reply]
One option it to connect the buck converter directly to PSU. I haven't done it myself but it seems some people are doing it. That way it turns on with the machine and it does not slow down.

**Reply by Sonicboom247** [reply]
@BlueWings172 
hmm, i connected it to the hotend fan terminals since that one is always on and not controllable...

**Reply by Sonicboom247** [reply]
Definitely need some taller feet now..

![20220213_131955](https://user-images.githubusercontent.com/84334417/153768993-60dfadf0-9390-4f69-ae20-e895469bb3bb.jpg)

**Comment by BlueWings172** [comment]
I'm planning to upgrade all my fans as well. Below is what I have in mind. I'm going with the Noctua 92 for the motherboard and PSU because it requires less clearance (14 vs 25 mm). I'm also planning to design a new motherboard case that can be accessed from the top as I have my printer inside an enclosure and I have to take it out every time I need to access the motherboard which drives my crazy.

@Cal-LF why do you have a fan for the Y stepper ? Does it get that hot ?



![Untitled](https://user-images.githubusercontent.com/65966061/153724098-ff6cbf81-80ff-4ffa-adc3-4f7ffdfa6f14.png)

**Reply by Cal-LF** [reply]
@BlueWings172 The Y motor vref is 1.0v and is still too hot for my liking. Temp is well within rating but 55c is is still to hot to the touch. The fan brings it to the same level as the other steppers which are warm to the touch.

**Reply by Sonicboom247** [reply]
@BlueWings172 great info! i like the organization..lol i was looking at the WINSINN fans and they did not get great reviews., the usual quality issues and lots of DOA's, Sunon Maglevs and Noctuas are the way to go.  Which hotend mount are you going to stuff those fans in? I already have the Avalon hotend printed which houses two 4020 fans but i am really trying to avoid Buck converters if i can.  I am going back and forth with Noctua in emails pushing for them to get more into 24v fans and corner the market.

**Reply by Sonicboom247** [reply]
Makes no sense, so creality connected both Fan1and Fan 2 together? I don't even understand the reasoning behind it...lol
![16447105321707613861541919888455](https://user-images.githubusercontent.com/84334417/153732733-91502b9b-d289-4317-bd68-a74cd1a4cbe7.jpg)

**Reply by BlueWings172** [reply]
> 
@Sonicboom247 
I have noticed that not all WINSINN  reviews are positive (5 stars at 70%) but the pack includes 4 fans so if anyone dies, I still have spares. After much reading, I've decided that I will only use radial fans for part cooling. What better radial fan options for part cooling are there ? 

I'm trying to avoid buck converter as well but 1) i'm only gonna use them for the motherboard, which as I understand is not PWM'ed and 2) I really like that the Noctua 92 is only 14mm thick. I'm designing a new box for the motherboard that can be opened from the top so I want to keep the thickness at minimum.

It is a shame that Noctua won't make 24V fans in more sizes and if they decided to today, it will take them months to have a product in the market.

**Comment by BlueWings172** [comment]
@Cal-LF , good to know. I haven't checked the temp on mine. Was thinking to upgrade the stepper to a 0.9 degree one. Do you have a link to the heatsink you are using ?

**Reply by BlueWings172** [reply]
Thanks. I do have dual motor for Z but I haven't touched Vref for any motor. Wonder if I should.

Seems like there is an advantage for having 0.9 steppers and I was looking at the E3D steppers but they are much longer than creality's. 

Have you used or heard of any other option ?

**Reply by Sonicboom247** [reply]
> For v422, vRef for X, Y -> 0.8v to 1.08v ; Z -> 0.95v to 1.1 v (1.22v, if you use 2 steps motors for DualZ with only 1 TMC); E -> 1.22v For v427, vRefs max for X,Y,Z -> 0.9v (1.1v with 2 steps motors for DualZ with only 1 TMC); E -> 1.1v. (based for the Creality Stock steps motors 42-34 et 42-40). If you use 0.9º step motors, the characteristics of the motors will be needed to recalculate the vRefs max. It will be necessary to multiply by 2 the steps/mm. The advantage being more torque and as the step motors are doubled, more precision.

wow, just checked mine and i am X,Y,Z: 1.2 -1.22V E: 1.39V, no wonder i cook eggs on my steppers...

**Reply by Sonicboom247** [reply]
so far these are my new settings i am testing vRef for X 945 @ 89 degrees F / Y 919 @ 101f /  Z 950 @ 101f / E 1191 @107.5f  41 min into a print, wish i had measured before changing them but compared to my Artillery Hornet, temperature wise the Ender is still running  about 10 degrees hotter

**Comment by Cal-LF** [comment]
@BlueWings172 I have no heatsink on the stepper, not needed with the fan.

**Reply by jurgendeleeuw** [reply]
I replaced my hotend one with 40 x 40 x 20 sunon 24 volt.also motherboard 80 x 80 x 25 sunon 24 volt.motherboard fan is connected toghether with hotend fan so always on.actually the sound is now half so loud so actually quite happy with the change 😁

**Reply by Sonicboom247** [reply]
i'm now looking at 4020 fans since i no longer run the stock hotend cover and can fit bigger fans.

**Comment by memomedfish** [comment]
Here is my question for the ones who changed the parts cooling fan.
I am using a Sunon 24v (MF40202V2-1000U-A99) and the voltage control from creality 4.2.7 board is not good at all. The fan either pulsates and work at around 20% or 100%. Did you figure out a way to control this issue?

**Reply by Sonicboom247** [reply]
if you have adaptive fan control enabled in firmware then that is perfectly normal.
#define ADAPTIVE_FAN_SLOWING              // Slow part cooling fan if temperature drops
your slicer can also ramp your fans up and down depending on how you have it set, but usually first few layers will have fan off and slowly ramp fans up to full as print goes. Again this is depending on filament and slicer, for example, PrusaSlicer will barely run your fan at all if you select PET filament.

**Comment by slepkaviba** [comment]
I  am still waiting for all parts to arrive but I used 120x15 on the psu , will use 80x25 on the motherboard and 40x20 for hotend.

Main goal is silent operation (as printer is in same room where I work/relax).

I also want to replace part cooling from 40x10 with 50x15  which I hope can cool better at lower speeds (so hopefully more silent).
